testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/client/AbstractClientTest.java

```diff
@@ -103,7 +103,7 @@ protected String createOidcBearerOnlyClient(String name) {
 
     protected String createOidcBearerOnlyClientWithAuthz(String name) {
         ClientRepresentation clientRep = createOidcClientRep(name);
-        clientRep.setBearerOnly(Boolean.TRUE);
+        clientRep.setBearerOnly(Boolean.FALSE);
```

Comment:
Should this method be renamed to something else? The name "createOidcBearerOnlyClientWithAuthz" is not appropriate now as created client won't be bearer-only.

Reply:
But the client is still confidential (the only type supported by authz)

Reply:
yeah, I know it is confidential. Hence my point to rename the method as it is a bit confusing now to have "bearerOnly" in the name since the client is not really bearer-only. How about "createOidcConfidentialClientWithAuthz" ?

Reply:
Sorry, Marek. I missed the reference in the method name.

Reply:
@mposolda, should be fixed now. Resolving the conversation. Sorry for missing that part.